cuchem/cuchem/utils/distance.py

```diff
@@ -16,7 +16,12 @@
 
 import math
 
-import cupy
+try:
+    import cupy as xpy
+except ModuleNotFoundError as e:
+    import numpy as xpy
```

Comment:
Same concern -- will users think numpy is supported if th epython imports are left as-is?

Reply:
I left it like this because we decided to leave CPU support as is, instead of spending more time and revisit later.

Reply:
Understood. We can clarify later if it comes up with users.